test-requirements.txt

```diff
@@ -3,8 +3,8 @@
 # process, which may cause wedges in the gate later.
 
 hacking<0.12,>=0.11.0 # Apache-2.0
-sphinx!=1.3b1,<1.4,>=1.2.1 # BSD
+sphinx>=1.5.1 # BSD
 mock>=2.0 # BSD
-coverage>=3.6 # Apache-2.0
-reno>=2.0.3 # Apache-2.0
 subunit2sql>=1.8.0
+coverage>=4.0 # Apache-2.0
+reno>=1.8.0 # Apache-2.0
```

Comment:
Here's another openstack global-requirements leak, but it's not a big deal. We really don't use reno for real yet so it's not like we really need it >=2.0.3